docs/_include/general-shipping/k8s.md

```diff
@@ -96,9 +96,25 @@ For a parameter called `someField` in the `logzio-logs-collector`'s `values.yaml
 --set logzio-logs-collector.someField="my new value"
 ```
 
-
 Adding `log_type` annotation with a custom value will be parsed into a `log_type` field with the same value.
 
+#### Default Log Level Detection
```

Comment:
I accidentally added a change from a different pr to the docs/_include/general-shipping/k8s.md page 🤦‍♀️ You can ignore it, there's another PR (#869) with the exact same changes